src/console-wrapper.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+const { HOME_DIR } = require('./constants');
+const fs = require('fs');
+const path = require('path');
+const stream = require('stream');
+const child_process = require('child_process');
+
+(async () => {
+    const fd = fs.openSync(path.join(HOME_DIR, '/reports/testcafe.log'), 'w+', 0644);
```

Comment:
I'd recommend calling this something more generic (a name that we can use across all frameworks). Like `console.log` or something else. The idea is that we'd auto select this file as a default view in our UI. If the name is different for each framework, that'd make it somewhat tougher to do 😅

Reply:
That would be better now 😄